modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/register/impl/RegisterServiceImpl.java

```diff
@@ -283,6 +290,8 @@ private RegisteredServiceVo convertToRegisteredServiceVo(RegisterServiceEntity s
         }
         RegisteredServiceVo registeredServiceVo = new RegisteredServiceVo();
         BeanUtils.copyProperties(serviceEntity, registeredServiceVo);
+        registeredServiceVo.add(
+                Link.of(openApiUtil.getOpenApiUrl(serviceEntity.getId().toString()), "openAPI"));
```

Comment:
something is wrong with the current configuration.. The field is shown as "_links" but when we do an actual GET.. only "links" is returned.

![image](https://user-images.githubusercontent.com/54129659/230902958-a605605f-3988-417a-8e77-1b78961a38bf.png)

Reply:
This error no longer exists  using field field 'openApiUrl' to return url.